.github/workflows/ci.yml

```diff
@@ -1,6 +1,6 @@
 name: CI
 
-on: pull_request
+on: [push, pull_request]
```

Comment:
This file is synchronized using our modulesync process, so please don't modify this file. Source file is here: https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.github/workflows/ci.yml.erb